src/python/WMComponent/JobAccountant/AccountantWorker.py

```diff
@@ -590,6 +590,9 @@ def associateLogCollectToParentJobsInWMStats(self, fwkJobReport, logAchiveLFN, t
 
         Associate a logArchive output to its parent job
         """
+        if self.fwjrCouchDB is None:
```

Comment:
I still think this belongs to the init method. And with the changes to the WorkerThread, this is not needed anymore, is it?

Reply:
I was just wondering whether connection failure is temporary problem. If it fails here the component crashes. I thought if it fails in the method it would be retried. I have to double check on that.